src/lib/java_hacks.js

```diff
@@ -0,0 +1,32 @@
+foam.CLASS({
```

Comment:
I would rather not add a file named java_hacks.js

Additionally are these changes necessary?  I fixed the issue with booleans not serializing properly and mentioned it to you on Jan 11.